packages/manager/modules/web-paas/src/components/delete-user/controller.js

```diff
@@ -2,16 +2,31 @@ import get from 'lodash/get';
 
 export default class {
   /* @ngInject */
-  constructor($translate, WebPaas) {
+  constructor($translate, atInternet, WebPaas) {
     this.$translate = $translate;
+    this.atInternet = atInternet;
     this.webPaasService = WebPaas;
   }
 
   $onInit() {
     this.isLoading = false;
   }
 
+  cancel() {
+    this.atInternet.trackClick({
+      name:
+        'web-paas::project-user-license::user-table-options::delete-user-cancel',
+      type: 'action',
+    });
+    return this.goBack();
+  }
+
   delete() {
+    this.atInternet.trackClick({
+      name:
+        'web-paas::project-user-license::user-table-options::delete-user-confirm',
```

Comment:
Can you extract common prefix and reuse it in all places. Refer kubernetes for code
/manager/modules/pci/src/projects/project/kubernetes/kubernetes.routing.js

kubeTrackPrefix: () => 'PublicCloud::pci::projects::project::kubernetes',

      sendKubeTrack: /* @ngInject */ (kubeTrackPrefix, trackClick) => (
        complement,
      ) => {
        trackClick(`${kubeTrackPrefix}::${complement}`);
      },

      trackClick: /* @ngInject */ (atInternet) => (hit) => {
        atInternet.trackClick({
          name: hit,
          type: 'action',
        });
      },